src/common/telemetry/src/logging.rs

```diff
@@ -43,6 +45,7 @@ pub struct LoggingOptions {
     pub enable_otlp_tracing: bool,
     pub otlp_endpoint: Option<String>,
     pub tracing_sample_ratio: Option<f64>,
+    pub remote_write: RemoteWriteOptions,
```

Comment:
Logging = log + tracing + metric?

Reply:
The original meaning is that there are some configurations related to observability in `logging`. Putting them here can also maximize the concentration of call points. 

How about :
1. make `remote_write` independent
2. or partially modify the configuration under `logging` , turn some observability-related configurations into a new configuration option called `observability` carrying with `remote_write`.

Reply:
Maybe make `remote_write` independent is more reasonable because of backward compatibility

Reply:
This is just a playful comment; I mistook this PR as one from @waynexia. We can just respect the tradition: it's acceptable for 'Logging' to stand for `observability`.